### PR TITLE
Connect Q/A (2)

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/aredotna/ervell/issues/545

I'm moving that last dropdown task to a new issue as it's somewhat larger in scope and unrelated.

------

------

### CODE REVIEW

**Comment by broskoski** [comment]
I love that your recent PR's have been net-negative in terms of line count

**Comment by broskoski** [comment]
And yeah, fine to move that issue out 👍

File: components/ui/buttons/overlay.styl [hunk 1]
```diff
@@ -1,9 +1,7 @@
 .Button
   &--overlay
     // Fit 2-up on block hover
-    extra-padding = 0.125em
-    padding (button-v-padding + extra-padding) (button-h-padding + extra-padding)
-    min-width (block_width / 2.25)
+    min-width (block_width / 2.42)
```
Comment (broskoski) [inline components/ui/buttons/overlay.styl]: 〰️

**Comment by broskoski** [comment]
LGTM, merge when ready

File: components/ui/buttons/overlay.styl (hunk 1 above)
Comment (dzucconi) [inline components/ui/buttons/overlay.styl]: `130px` I like round numbers too but

**Pull request merged**